lib/dpul_collections_web/live/search_live.ex

```diff
@@ -343,4 +343,28 @@ defmodule DpulCollectionsWeb.SearchLive do
       type == :post -> [{"...", false}, {page, false}]
     end
   end
+
+  defp thumbnail_service_urls(max_thumbnails, item) do
+    thumbnail_service_urls(
+      max_thumbnails,
+      item.image_service_urls,
+      item.primary_thumbnail_service_url
+    )
+  end
+
+  defp thumbnail_service_urls(max_thumbnails, image_service_urls, nil) do
+    # Truncate image service urls to max value
+    image_service_urls
+    |> Enum.slice(0, max_thumbnails)
```

Comment:
:joy: Yeah I was wondering about this, were we just putting every thumbnail on the page? I guess it was performing.